cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -587,4 +603,77 @@ describe('Top toolbar tests.', function() {
 		}).then(() => expect(data).to.deep.eq(expectedData));
 
 	});
+
+	it('Apply superscript.', function() {
```

Comment:
if we are testing the top toolbar shouldn't we use buttons from the top toolbar for this and all the tests below?

Reply:
superscript and subscripts buttons are not there for classic menu